docs/application/native/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1174 @@
+  The default value of recognition mode is `VC_RECOGNITION_MODE_STOP_BY_SILENCE`. If you want to set the manual mode, you can use `VC_RECOGNITION_MODE_MANUAL`.
+  These functions are used when the voice control manager is in the `VC_STATE_READY` state:
+
+  ```c
+  /* Get */
+  void get_recognition_mode()
+  {
+      int ret;
+      vc_recognition_mode_e mode;
+      ret = vc_mgr_get_recognition_mode(&mode);
+      if (VC_ERROR_NONE != ret)
+          /* Error handling */
+  }
+
+  /* Set */
+  void set_recognition_mode(vc_recognition_mode_e mode)
+  {
+      int ret;
+      ret = vc_mgr_set_recognition_mode(mode);
+      if (VC_ERROR_NONE != ret)
+          /* Error handling */
+  }
+  ```
+
+- Get or set private data between voice control manager and voice control engine using the `vc_mgr_get_private_data()` and `vc_mgr_set_private_data()` functions.
+  `vc_mgr_get_private_data()` is used when the parameters move from voice control engine to voice control manager.
+  `vc_mgr_set_private_data()` is used when the parameters move from voice control manager to voice control engine.
+  These functions are used when the voice control manager is in the `VC_STATE_READY` state:
+
+  ```c
+  /* Get */
+  void get_private_data(const char* private_key)
+  {
+      int ret;
+      char* data = NULL;
+      ret = vc_mgr_get_private_data(private_key, &data);
+      if (VC_ERROR_NONE != ret)
+          /* Error handling */
+  }
+
+  /* Set */
+  void set_private_data(const char* private_key, const char* private_data)
+  {
+      int ret;
+      ret = vc_mgr_set_private_data(private_key, private_data);
+      if (VC_ERROR_NONE != ret)
+          /* Error handling */
+  }
+  ```
+
+
+<a name="commands"></a>
+## Managing commands
+
+You can use command list to manage the commands. You can add or remove the commands to the command list and retrieve the command information using the command list.
+
+To create a command list and commands:
+
+1.  Create a command list with a command list handle.
+    The command list can have multiple commands. Each command has a text and a type.
+    The group can have `VC_COMMAND_TYPE_BACKGROUND`, `VC_COMMAND_TYPE_EXCLUSIVE`, `VC_COMMAND_TYPE_FOREGROUND`, `VC_COMMAND_TYPE_SYSTEM`, `VC_COMMAND_TYPE_SYSTEM_BACKGROUND`, and `VC_COMMAND_TYPE_WIDGET` type commands.
+
+    > [!NOTE]
+    > The order of command group priority is `VC_COMMAND_TYPE_SYSTEM`, `VC_COMMAND_TYPE_WIDGET`, `VC_COMMAND_TYPE_FOREGROUND`, `VC_COMMAND_TYPE_SYSTEM_BACKGROUND`, and `VC_COMMAND_TYPE_BACKGROUND`. The `VC_COMMAND_TYPE_EXCLUSIVE` is used on special situation.
+    > Normally, the `VC_COMMAND_TYPE_SYSTEM` and `VC_COMMAND_TYPE_SYSTEM_BACKGROUND` commands are only registered in the voice control manager.
+    > The `VC_COMMAND_TYPE_FOREGROUND` and `VC_COMMAND_TYPE_BACKGROUND` commands are used in the voice control client application.
+    > The `VC_COMMAND_TYPE_WIDGET` command is automatically registered in the elementary on the screen.
```

Comment:
Not clear. 
What is elementary?

Reply:
Elementary means the UI object like button, check box, text label, and so on.
https://docs.tizen.org/application/native/api/mobile/6.0/group__Elementary.html